<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>184</width>
    <height>134</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="addFileButton">
      <property name="toolTip">
       <string>Add a single volume or image</string>
      </property>
      <property name="text">
       <string>Add File</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../gui/icons/32x32/actions/media-seek-forward.png</normaloff>../../../gui/icons/32x32/actions/media-seek-forward.png</iconset>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="addStackButton">
      <property name="toolTip">
       <string>Create a single 3D dataset in your project from a directory of images files</string>
      </property>
      <property name="text">
       <string>Import Stack Directory</string>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="addStackFilesButton">
      <property name="toolTip">
       <string>Select several files to create a single 3D stack</string>
      </property>
      <property name="text">
       <string>Import Stack Files...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="removeFileButton">
      <property name="toolTip">
       <string>Remove the selected dataset from your project</string>
      </property>
      <property name="text">
       <string>Remove</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../gui/icons/32x32/actions/media-seek-backward.png</normaloff>../../../gui/icons/32x32/actions/media-seek-backward.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
